### DISCUSSION TITLE
Make HTMX native rather than React, Vue, etc.

### DISCUSSION THREAD

**Discussion by 18a93a664c** [livebud/bud#51] [Ideas]
Think about integrating htmx as a native front-end module. Also rather than just postgres, include sqlite as a default option

**Comment by matthewmueller** [comment]
Thanks for your comment @18a93a664c! Are you talking about this project? https://htmx.org/. Can you share what you like about htmx?

> Also rather than just postgres, include sqlite as a default option

Yes, this will happen.

**Reply by 18a93a664c** [reply]
Yes, that is what I was referring to...the attraction to projects like that have been raised in other discussions. Straight HTML being sent to the client. At this point in my dev experience, I'm actively removing any overhead (context switching) that I can. I'm burnt out on the JS zeitgeist, and just recently began learning golang. Being able to find a framework that is just basically Go on the backend with HTML on the frontend would be near perfection.

I read the HN thread and saw that Pagoda might be a bit more what I'm looking for out of the box.
